projects/pastanaga-angular/src/lib/styles/_buttons.scss

```diff
@@ -39,7 +39,9 @@
         font-size: font-size(s);
         line-height: rhythm(3);
         padding: $padding-button-small;
-
+        &:not(.pa-button-icon):not(.pa-button-icon-text) {
+            min-width: auto;
+        }
```

Comment:
would be better to add a condition `:not(.pa-small)` where we set the min-width instead (line 34).

Reply:
done